web/admin.py

```
@@ -623,6 +624,38 @@ class LearningStreakAdmin(admin.ModelAdmin):
     search_fields = ("user__username",)
 
 
+
+@admin.register(SocialShareDiscount)
+class SocialShareDiscountAdmin(admin.ModelAdmin):
+    list_display = ("user", "course", "platform", "status", "discount_amount", "created_at", "expires_at")
+    list_filter = ("status", "platform", "created_at")
+    search_fields = ("user__username", "user__email", "course__title", "share_url")
+    readonly_fields = ("verification_token", "created_at", "verified_at", "expires_at", "used_at")
+    actions = ["mark_as_verified", "mark_as_expired"]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)